Assets/Scripts/Gameplay/Player/PlayerStats.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerStats : MonoBehaviour
+{
+    BaseStuct test = new BaseStuct();
+
+    public float MaxHP;
+    public float CurrentHP;
+    public float PlayerSpeed;
+    //public float PlayerLuck = 100.0f;
```

Comment:
ideally these should be encapsulated with getters and setters to prevent potential abuse later in the code base.